packages/moonstone/EditableIntegerPicker/EditableIntegerPicker.js

```diff
@@ -113,16 +121,16 @@ const EditableIntegerPickerBase = kind({
 		inputRef: PropTypes.func,
 
 		/**
-		 * A function to be run when there is a blur in the input
+		 * Called when there is a blur in the input.
 		 *
 		 * @type {Function}
 		 * @public
 		 */
 		onInputBlur : PropTypes.func,
 
 		/**
-		 * A function to be run when pickerItem is clicked.
-		 * This enables the input field in the component
+		 * Called when the pickerItem is clicked.
+		 * This enables the input field in the component.
```

Comment:
Perhaps this should say that the user should enable the input in response to this event?